### DISCUSSION TITLE
Top Icon Row

### DISCUSSION THREAD

**Discussion by KevDoy** [PintaProject/Pinta#892] [General]
I'm using Pinta on macOS, and I love it. As someone who came from Paint.net many years ago, Pinta is a great tool for simple jobs when I'm not looking to open up Photoshop.

However, as a UX/UI designer, I would ideally like to change the UI ever so slightly. I have a suggestion:

Here is what Pinta's top icon row looks like on macOS today:
![image](https://github.com/PintaProject/Pinta/assets/11251011/07db32df-31de-4966-b6bf-0ff9924e9eca)

Here's a slightly cleaner, smaller variety utilizing the Tango! Icon replacements from ysUI that I've been working on (https://github.com/ysUI-org)
![image](https://github.com/PintaProject/Pinta/assets/11251011/27add6ce-badd-4a71-a032-f288969b0d02)

This brings, at least the top, into alignment with macOS a little more. 

Would love your thoughts.

**Comment by cameronwhite** [comment]
There have been some changes in the `master` branch with the port to GTK4 / libadwaita, which looks like this now:

<img width="709" alt="Screenshot 2024-06-30 at 10 10 20 AM" src="https://github.com/PintaProject/Pinta/assets/303951/3b9473ea-c1ea-4848-9ddd-d64cd1cd56c9">

I don't object to shipping custom icons if it's just for macOS, where we're already bundling GTK and an icon theme with Pinta. But for Linux I think users would expect to see their specific GTK icon theme being used for standard icons

**Reply by KevDoy** [reply]
That makes total sense. Appreciate the update to the icon sizing here!

Yes, the icons are purely to bring Pinta more in-line with the macOS/Apple visual theme. Happy to provide zip file with svgs, or to open a pull request if I know where to put them.

Cheers

**Reply by cameronwhite** [reply]
A zip file is probably fine, I think this would need some other changes in the build system to have them properly bundled for macOS but not other platforms

**Reply by KevDoy** [reply]
That makes sense. I've attached the icons I used below. Let me know if you need anything else!

**[ysui-icons-pinta.zip](https://github.com/user-attachments/files/16078222/ysui-icons-pinta.zip)**

_source: https://github.com/ysUI-org/ysUI-Icons/_

**Reply by cameronwhite** [reply]
Thanks! I've transferred this to #908 so that it's in the issue tracker
